ast/src/analyzed/mod.rs

```diff
@@ -490,6 +490,7 @@ pub struct Symbol {
     pub id: u64,
     pub source: SourceRef,
     pub absolute_name: String,
+    pub stage: Option<u32>,
```

Comment:
```suggestion
    pub stage: u32,
```

Is a stage of `None` the same as `Some(0)`?

Reply:
Oh I guess you'd have a stage iff. `kind == SymbolKind::Poly(PolynomialType::Committed)`? I think it might still make sense to assign stage 0 for all other symbols, because they are usable in all stages?